message.go

```diff
@@ -1236,3 +1260,44 @@ func (m *RedisMessage) approximateSize() (s int) {
 	}
 	return
 }
+
+// String returns human-readable representation of RedisMessage
+func (m RedisMessage) String() string {
+	v, err := m.MarshalJSON()
+	if err != nil {
+		return ""
+	}
+	return string(v)
+}
+
+// MarshalJSON implements json.Marshaler interface
+func (m RedisMessage) MarshalJSON() ([]byte, error) {
+	type PrettyRedisMessage struct {
+		Type  string `json:"Type,omitempty"`
+		Error string `json:"Error,omitempty"`
+		Ttl   string `json:"Ttl,omitempty"`
+		Value any    `json:"Value,omitempty"`
+	}
+	strType, ok := typeNames[m.typ]
+	if !ok {
+		strType = "unknown"
+	}
+	obj := PrettyRedisMessage{Type: strType}
+	if m.ttl != [7]byte{} {
+		obj.Ttl = time.UnixMilli(m.CachePXAT()).UTC().String()
+	}
+	if err := m.Error(); err != nil {
+		obj.Error = err.Error()
+	}
+	switch m.typ {
+	case typeFloat, typeBlobString, typeSimpleString, typeVerbatimString, typeBigNumber:
+		obj.Value = m.string
+	case typeBool:
+		obj.Value = strconv.FormatBool(m.integer == 1)
```

Comment:
Couldn't it just be like this?
```suggestion
		obj.Value = m.integer == 1
```